.gitignore

```diff
@@ -25,3 +25,5 @@ debian/pylint
 .eggs/
 .pytest_cache/
 .mypy_cache/
+.vscode
```

Comment:
This should probably go in your global `.gitignore` (ie : `git config --global core.excludesfile ~/.gitignore `)

Reply:
Resolving based on earlier conversation:
https://github.com/PyCQA/pylint/pull/3769#discussion_r484534292